src/components/ChartRow.js

```diff
@@ -151,18 +144,31 @@ export default class ChartRow extends React.Component {
 
                 const scale = createScale(child, type, min, max, rangeBottom, rangeTop);
                 if (!_.has(this.scaleMap, id)) {
-                    // No scale map yet, create one on this.state.yAxisScalarMap
+                    // If necessary, initialize a ScaleInterpolator for this y-axis.
```

Comment:
For line 139-142, you can replace the if condition with the `isChildYAxis(child)`

Reply:
Same here as above.